02-express-tutorial/public/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Product List</h1>
+    <!-- <button id="fetchProductsBtn">Fetch Products</button>
+    <div id="productList"></div> -->
+
+    <script src="../app.js"></script>
```

Comment:
Quick hint if you'd like to try to get question 4 to work: use [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) to call your API instead of importing the script here.